docs/json/radarr/cf/dv-hlg.json

```diff
@@ -14,12 +14,12 @@
       }
     },
     {
-      "name": "Not DV HDR10",
+      "name": "Not HDR10",
       "implementation": "ReleaseTitleSpecification",
       "negate": true,
       "required": true,
       "fields": {
-        "value": "^(?=.*\\b(DV|DoVi|Dolby[ .]?Vision)\\b)(?=.*\\b(HDR(10)?(?!\\+))\\b)"
+        "value": "\\b(HDR(10)?(?!\\+))\\b"
```

Comment:
Will in a different PR change the regex for HDR10 to be consistent across all formats